http_client.rst

```diff
@@ -627,6 +627,11 @@ according to the ``multipart/form-data`` content-type. The
         'body' => $formData->bodyToIterable(),
     ]);
 
+By default, :class:`Symfony\\Component\\Mime\\Part\\Multipart\\FormDataPart`
+will transfer file uploads in binary encoding. But you can enforce Base64::
+
+    $fileField = DataPart::fromPath('/path/to/uploaded/file', null, null, 'base64');
+
```

Comment:
```suggestion

.. versionadded:: 6.3
    The `encoding` parameter was added to the `fromPath()` method in Symfony 6.3.

```

Reply:
The encoding parameter was introduced in Symfony 6.3.



Should be enough 👍🏻